.changes/v0.19.8.md

```diff
@@ -0,0 +1,40 @@
+## v0.19.8 - 2025-12-04
+
+### Added
+- Optimize `WithDirectory` and `WithFile` to avoid copying file data on disk in more cases by @sipsma in https://github.com/dagger/dagger/pull/11497 \
+  These APIs now avoid copying data on disk in all cases except when an owner, permission or timestamp is modified as part of the operation.
+- Add support for host environment variable expansion in `.env` files by @alexcb in https://github.com/dagger/dagger/pull/11442
+- Add VCS support for on-premises Azure DevOps by @nilune in https://github.com/dagger/dagger/pull/11500
+- Add ability to set app role mount path in Vault secret provider by @nilune in https://github.com/dagger/dagger/pull/11471
+- Add toolchain customization configuration by @kpenfound in https://github.com/dagger/dagger/pull/11480
+- Add configuration to ignore checks in a toolchain by @kpenfound in https://github.com/dagger/dagger/pull/11489
+- TypeScript SDK: Add live telemetry support by @TomChv in https://github.com/dagger/dagger/pull/11486
+- TypeScript SDK: Add tracing for tests using `@otr/mocha-test` lib by @TomChv in https://github.com/dagger/dagger/pull/11508
+- Add `File.AsJSON()` to simplify handling JSON file contents by @suprjinx in https://github.com/dagger/dagger/pull/11408
+
+### Changed
+- Break up CI monolith into cleanly separated toolchains by @shykes, @kpenfound, @eunomie and @TomChv in https://github.com/dagger/dagger/pull/11373
```

Comment:
Not sure if CI changes are relevant to release notes for users (not a blocker though)